packages/nodes-base/nodes/Google/Docs/DocumentDescription.ts

```diff
@@ -0,0 +1,825 @@
+										description: 'Inserts the text at the end of a header, footer, footnote or the document body.',
+									},
+									{
+										name: 'Location',
+										value: 'location',
+										description: 'Inserts the text at a specific index in the document.',
+									},
+								],
+								description: 'The location where the text will be inserted.',
+								default: '',
+							},
+							{
+								displayName: 'Segment ID',
+								name: 'segmentId',
+								type: 'string',
+								description: 'The ID of the header, footer or footnote the location is in. An empty segment ID signifies the document\'s body.',
+								default: '',
+							},
+							{
+								displayName: 'Index',
+								name: 'index',
+								type: 'number',
+								description: 'The zero-based index, relative to the beginning of the specified segment.',
+								displayOptions: {
+									show: {
+										locationChoice: [
+											'location',
+										],
+									},
+								},
+								default: '',
+							},
+						],
+					},
+					{
+						name: 'createParagraphBulletsValues',
+						displayName: 'Create Paragraph Bullets Style',
+						values: [
+							{
+								displayName: 'Bullet Preset',
+								name: 'bulletPreset',
+								type: 'options',
+								options: [
+									{
+										name: 'Bullet glyph preset unspecified',
+										value: 'BULLET_GLYPH_PRESET_UNSPECIFIED',
+										description: 'The bullet glyph preset is unspecified.',
+									},
+									{
+										name: 'Bullet disc circle square',
+										value: 'BULLET_DISC_CIRCLE_SQUARE',
+										description: 'A bulleted list with a DISC , CIRCLE and SQUARE bullet glyph for the first 3 list nesting levels.',
+									},
+									{
+										name: 'Bullet diamondx arrow3d square',
+										value: 'BULLET_DIAMONDX_ARROW3D_SQUARE',
+										description: 'A bulleted list with a DIAMONDX , ARROW3D and SQUARE bullet glyph for the first 3 list nesting levels.',
+									},
+									{
+										name: 'Bullet checkbox',
+										value: 'BULLET_CHECKBOX',
+										description: 'A bulleted list with CHECKBOX bullet glyphs for all list nesting levels.',
+									},
+									{
+										name: 'Bullet arrow diamond disc',
+										value: 'BULLET_ARROW_DIAMOND_DISC',
+										description: 'A bulleted list with a ARROW , DIAMOND and DISC bullet glyph for the first 3 list nesting levels.',
+									},
+									{
+										name: 'Bullet star circle square',
+										value: 'BULLET_STAR_CIRCLE_SQUARE',
+										description: 'A bulleted list with a STAR , CIRCLE and SQUARE bullet glyph for the first 3 list nesting levels.',
+									},
+									{
+										name: 'Bullet arrow3d circle square',
+										value: 'BULLET_ARROW3D_CIRCLE_SQUARE',
+										description: 'A bulleted list with a ARROW3D , CIRCLE and SQUARE bullet glyph for the first 3 list nesting levels.',
+									},
+									{
+										name: 'Bullet lefttriangle diamond disc',
```

Comment:
An idea for readability of options like this one:
```suggestion
										name: 'Bullet glyphs: left triangle, diamond, disc',
```